src/lib/modules/tests/test.js

```diff
@@ -370,11 +360,11 @@ class Test {
       newContract.options.gas = constants.tests.gasLimit;
     }
 
-    return newContract;
+    cb(null, newContract);
```

Comment:
You made this function async, but in the end, it's not actually needed. Do you think we should put it as before or the callback might be needed at some point.

Reply:
Yeah it's strangely interesting. I tried removing the callback, and the tests just refuse to pass for reasons unknown at this point.